docs/user-guide/howto.rst

```diff
@@ -157,6 +157,20 @@ one for each time bin.
 
 .. accordion-footer::
 
+.. accordion-header::
+    :id: collapseHowToOne
+    :title: Non-convergence of a fit because of IRFs validity range
+
+It happens that a 3D analysis does not converge with warning messages indicating that the
+scanned positions of the model is outside valid IRF map range. This issue might happen when
+the position of a model has no defined range. The minimizer might scan positions outside the
+spatial range in which the IRFs are computed and then it gets lost.
+
+The simple solution is to add a physically-motivated range on the model's position, which
```

Comment:
Maybe also restrict the model position to the FoV size?

Reply:
I have hesitated to mention that. Indeed, Quentin have reminded that some extended source could have their center outside the FoV and their extension is such that a part lies into the FoV. In this case, this could work, no?
This is why I wrote this generic formulation...

Reply:
Makes sense, however one could still include some reasonable margin...